home/templatetags/home_tags.py

```diff
@@ -1,16 +1,17 @@
 from django import template
-from home.models import Footer
+import home.models as model
 
 register = template.Library()
 
 @register.inclusion_tag('home/tags/footer.html', takes_context=True)
 def footer(context):
     return {
-        'footer': Footer.objects.first(),
+        'footer': model.Footer.objects.first(),
         'request': context['request'],
     }
 
 
 @register.inclusion_tag('home/tags/articles_list.html')
 def render_articles_list(articles):
-    return {'articles': articles}
+    live_articles = articles.filter(live=True)
```

Comment:
I think we should alter filter these articles from the view vs the template tag. I haven't tested this, but [https://docs.wagtail.io/en/stable/reference/pages/queryset_reference.html](the documentation) seems to suggest that we can filter it by `Page.objects.live()...`

Reply:
It works, I tested it before pushing.

`template tag` has 4 loops.
[https://github.com/unicef/iogt/blob/336d94d7fc7ee4426962219aeb90ee9829c78f28/home/templates/home/tags/articles_list.html#L4]
[https://github.com/unicef/iogt/blob/336d94d7fc7ee4426962219aeb90ee9829c78f28/home/templates/home/tags/articles_list.html#L22]
[https://github.com/unicef/iogt/blob/336d94d7fc7ee4426962219aeb90ee9829c78f28/home/templates/home/tags/articles_list.html#L41]
[https://github.com/unicef/iogt/blob/336d94d7fc7ee4426962219aeb90ee9829c78f28/home/templates/home/tags/articles_list.html#L74]

In this case, I thought it's better to filter from view instead of template tag.

Reply:
I meant it's more 'standard' to filter the articles in the view. Business logic should not spill over into the templates. The templates should be as `dumb` as possible

Reply:
So can we keep current updates?

Reply:
I would say we should still filter the articles in the views and not in template tags

Reply:
I agree with @ehmadzubair. This is a rather simple case, so it does not seem to make much difference where the query goes, however, I suspect there may be more things to consider in the future than just whether articles are live or not, such as which order the articles should be in, and perhaps handling other items than articles.

We should also consider how easy it is to test, given that the home page will be more complicated later on. It would also be good to include a test that ensures that only live articles are present on the home page, @djangodeveloper96 .